Editor/AGS.Editor/Components/TranslationsComponent.cs

```diff
@@ -216,7 +220,10 @@ private void DoTranslationUpdate(IList<Translation> translations)
             _agsEditor.SaveGameFiles();
             CompileMessages errors = (CompileMessages)BusyDialog.Show("Please wait while the translation(s) are updated...", new BusyDialog.ProcessingHandler(UpdateTranslationsProcess), translations);
             _guiController.ShowOutputPanel(errors);
-            _guiController.ShowMessage("Translation(s) updated.", MessageBoxIcon.Information);
+            if (errors.Count == 0)
+                _guiController.ShowMessage("Translation(s) updated.", MessageBoxIcon.Information);
+            else
+                _guiController.ShowMessage("Translation(s) update failed.", MessageBoxIcon.Error);

```

Comment:
Maybe add something like "Check output panel for more information" here? because if output panel is not shown, then the user will wonder why the process has failed at all.